core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -773,6 +773,18 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     assertEquals(numRecords, endOffsets.get(tp))
   }
 
+  @ParameterizedTest(name = TestInfoUtils.TestWithParameterizedQuorumAndGroupProtocolNames)
+  @MethodSource(Array("getTestQuorumAndGroupProtocolParametersAll"))
+  def testSeekThrowsIllegalStateIfPartitionsNotAssigned(quorum: String, groupProtocol: String): Unit = {
+    val tp = new TopicPartition(topic, 0)
+    consumerConfig.put(ConsumerConfig.GROUP_ID_CONFIG, "my-group-id")
+    consumerConfig.put(ConsumerConfig.GROUP_INSTANCE_ID_CONFIG, "my-instance-id")
```

Comment:
do we need these 2 here? (I wouldn't expect so)

Reply:
No, we don't need to join any group so it can be removed.